lib/tictactoe/game.rb

```diff
@@ -71,25 +72,49 @@ def check_winner()
       players = ["o", "x"]
       players.each do |symbol|
         if winning_combinations(symbol)
-          puts "Player using '#{symbol}' has won!"
-          return true
+          return true, symbol
         end
       end
+      return false
+    end
+
+    def output_wining_message(symbol)
+      @io.output(@messages.get_message("player_wins", symbol))
+    end
+  
+    def play_again_input
+      @io.output(@messages.get_message("play_again"))
+      input=@io.input
+      if input == "1"
+        @game = true
+      end
+    end
+
+    def play_again()
+      while @game
+        play
+      end
     end
 
     def play()
       draw()
-      while @board.is_not_full
-        if check_winner() != true
-          symbol = get_symbol()
-          position = get_position()
-          make_move(position, symbol)
-          draw()
-        else
-          return false
-        end
+      while @board.is_not_full && !check_winner()
+        symbol = get_symbol()
+        position = get_position()
+        make_move(position, symbol)
+        draw()
+      end
+      @game = false
+      symbol = check_winner()
+      if symbol && symbol[0]==true
+        output_wining_message(symbol[1])
       end
-      puts " IT'S A DRAW!"
+      if !@board.is_not_full
```

Comment:
`!@board.is_not_full` is double negative, could you capture this logic in a more descriptive method to make this easier to read?